### DISCUSSION TITLE
Coding

### DISCUSSION THREAD

**Discussion by Miala-python** [ScratchAddons/ScratchAddons#3990] [Help] [answered]
Hello, I would like to help you for the creating of addons but I don't know programming in js.

Can you recommend to me a tutorial ?

**Comment by FunctionalMetatable** [comment, marked as answer]
~https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/~'

Learning JS:
https://www.w3schools.com/js/
https://www.javascript.com/learn/strings

**Reply by CST1229** [reply]
That is not a tutorial for programming in JS.

**Reply by FunctionalMetatable** [reply]
oh yeah i got confused

stick to debugging a svelte app for few weeks and look what has happened to me

**Reply by FunctionalMetatable** [reply]
i edited the post

**Reply by Miala-python** [reply]
> ~https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/~'
> 
> Learning JS:
> https://www.w3schools.com/js/
> https://www.javascript.com/learn/strings

Thank you very much

**Reply by nexensys** [reply]
What about https://javascript.info/ or the mdn web docs

**Reply by hello-smile6** [reply]
> What about https://javasript.info/ or the mdn web docs

The website you linked would be https://javascript.info/ , just pointing that out. Also, https://developer.mozilla.org/ is a good resource.

**Reply by cobaltt7** [reply]
> Also, [developer.mozilla.org](https://developer.mozilla.org/) is a good resource.

that's mdn web docs

**Reply by hello-smile6** [reply]
> > Also, [developer.mozilla.org](https://developer.mozilla.org/) is a good resource.
> 
> that's mdn web docs

It is!!!

**Comment by Miala-python** [comment]
Thanks anyway !
